microsoft/utils/aitl/aitl.py

```diff
@@ -103,7 +117,7 @@ def _add_resource_parser(
             action_parser, support_name=support_name, required_name=required_name
         )
 
-        if resource in {"job", "template"} and action == "create":
+        if resource in {"job", "template"} and action in {"create", "update", "patch"}:
```

Comment:
It should use`[ ]` as an array. `{ }` is used for dict, I'm not sure what type it is here.

Reply:
Please also fix build errors.